client/src/store/remote.ts

```diff
@@ -81,15 +81,15 @@ export const actions = actionTree(
     },
 
     request({ getters }) {
-      if (!accessor.connected || !getters.hosting) {
+      if (!accessor.connected || getters.hosting) {
```

Comment:
How comes this never to be spotted earlier? Everything works so far, is that value even correctly set?

Reply:
Oh, it seems like that function is not even used at all.